group_clusters.go

```diff
@@ -45,6 +45,7 @@ type GroupCluster struct {
 	User               *User               `json:"user"`
 	PlatformKubernetes *PlatformKubernetes `json:"platform_kubernetes"`
 	Group              *Group              `json:"group"`
+	ManagementProject  *ManagementProject  `json:"management_project"`
```

Comment:
I try to keep the ordering as it's in the docs. So this line should be placed above `Group`.

Reply:
Done